profile/README.md

```diff
@@ -0,0 +1,35 @@
+# Welcome to jQAssistant-Plugin 👋
+
+jQAssistant is an Open-Source-Tool that helps you to analyze and control the quality of your software systems. 
+Its plugin-based architecture allows the user to adapt jQAssistant to the needs of the analyzed project by adding
+* scanner plugins to support additional file formats
+* rule plugins to support frameworks, technologies, and libraries as well as common architectural and design patterns
+* report plugins to support the reporting of results in new formats
+
+## About the organization
+
+This GitHub organization provides a collection of useful extensions to jQAssistant.
+
+The core jQAssistant distribution is developed in our [main organization](https://github.com/jqassistant).
+Please have a look there for general information and when having issues or feature requests.
+
+Examples and tutorials can be found in our [examples organization](https://github.com/jqassistant-tutorials).
+We recommend that every (new) user takes a look at it and be inspired.
+
+## Way of contribution
+
+We're happy about every contribution. To make this as efficient as possible, take a look at the following guidance:
+
+* Issues should have a meaningful description, best with an example for reproduction. If possible to identify, put the issue into the repository of the part (e.g. core, a plugin, an example) where the issue occurred. This saves us a lot of time.
+  * We're happy to accept pull requests that solve the issue. 
+* Feature Requests should have a meaningful description, best with an example. If possible to identify, put the feature request into the repository of the part (e.g. core, a plugin, an example) where you expect the feature to be. This saves us a lot of time.
+    * We're happy to accept pull requests that implement the request.
+
+For any other topics, feel free to [Contact Us](mailto:info@jqassistant.com)
+
+## Further Information
+
+* [User Manual](https://jqassistant.github.io/jqassistant/doc/1.11.1/manual/)
```

Comment:
https://jqassistant.github.io/jqassistant/doc/ automatically forwards to the latest available documentation